internal/datacoord/garbage_collector.go

```diff
@@ -54,21 +56,24 @@ type garbageCollector struct {
 	meta     *meta
 	segRefer *SegmentReferenceManager
 
+	rootCoordClient types.RootCoord
```

Comment:
Go prefers shorter variable names, for example `rcc`.

Reply:
ok